docs/section22/Rule22-23.md

```diff
@@ -0,0 +1,52 @@
+
+# CHW&CW - Rule 22-23  
+
+**Rule ID:** 22-23  
+**Rule Description:** The baseline chiller efficiencies shall be modeled at the minimum efficiency levels, both part load and full load, in accordance with Tables G3.5.3.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.2.1 Equipment Efficiencies  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** Table G3.5.3  
+**Function Call:**  
+
+1. get_baseline_system_types()
+
+**Applicability Checks:**  
+
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.RulesetModelInstance.chillers:`
+
+  - Get chiller rated capacity: `rated_capacity_b = chiller_b.rated_capacity`
+
+    - If chiller rated capacity is less than 150Ton, set size category to < 150TONS: `if rated_capacity_b < 150 * CONVERSION(TON_TO_BTUH): size_category = "< 150TONS"`
+
+    - Else if chiller rated capacity is 150Ton or more and is less than 300Ton, set size category to >= 150TONS and < 300TONS: `else if rated_capacity_b < 300 * CONVERSION(TON_TO_BTUH): size_category = ">= 150TONS AND < 300TONS"`
+
+    - Else if chiller rated capacity is 300Ton or more and is less than 600Ton, set size category to >= 300TONS and < 600TONS": `else if rated_capacity_b < 600 * CONVERSION(TON_TO_BTUH): size_category = ">= 300TONS AND < 600TONS"`
+
+    - Else, set size category to > 600TONS: `else: size_category = "> 600TONS"`
+
+    - Get baseline chiller efficiency requirements: `kW_ton_full_load_b = data_lookup(table_G3_5_3, chiller_b.compressor_type, size_category, "FL"), kW_ton_part_load_b = data_lookup(table_G3_5_3, chiller_b.compressor_type, size_category, "IPLV")`
+
+      **Rule Assertion - Component:**
+
+      - Case 1: For each chiller, if chiller full-load and part-load efficiencies are modeled at the minimum efficiency levels in accordance with Tables G3.5.3 `if ( chiller_b.full_load_efficiency == kW_ton_full_load_b ) AND ( chiller_b.full_load_efficiency == part_load_efficiency ) AND ( chiller_b.part_load_efficiency_metric == "INTEGRATED_PART_LOAD_VALUE" ): PASS`
+
```

Comment:
1. chiller.full_load_efficiency is expressed in COP, while the values in table G3.5.3 are in kW/ton, so please add conversion to chiller_b.full_load_efficiency * CONVERSION (COP to kW_ton)==kW_ton_full_load_b
2. after AND should say (chiller_b.part_load_efficiency==...) (instead of full load)

Reply:
Updated.

Reply:
@mkarpman we had tacked the same discussion on efficient conversion when we evaluated boiler efficient rule and decided that is the RMR uses a value other than AFUE then the rule outcome would be fail. The logic there has been that the conversion from AFUE to Et isnt as straightforward. However, here we are doing a kW per ton to COP conversion - i assume thats because there is a standard protocol for this conversion? Please confirm

Reply:
Correct. AFUE reflects annualized average efficiency while Et is steady state thermal efficiency, so there is no physics-based conversion that is correct for any model of boiler. On the other hand, chiller.full_load_efficiency is expressed as COP (W/W) while ASRHAE tables have it as kW/Ton, so we need to apply conversion.

Reply:
Taking a fresh look at this RDS I think we should separate part load efficiency into a separate RDS that would be based on chiller.power_validation_points. IPLV is typically not entered / used by simulation tools, which would mean that majority of models would fail the RDS

Reply:
@mkarpman I believe the decision on the IPLV verification was to verify that the baseline chiller validation points align with the part load performance curves that are being added to App G 2022, is that correct?

Reply:
I thought the plan was to check the IPLV field via RDS and have an RMR test to verify that the tool converts either performance curve or validation points to the IPLV correctly.

Reply:
@mkarpman recommends splitting this RDS into two RDS. the first RDS will verify full load and part load efficiency. 

Regarding IPLV test:  We will check the absolute value of the IPLV through RDS and had curve or validation points to IPLV conversion checked via RMR test. 
Jeremy Lerond  will provide curves and table. For each set of curves and tables, there will be a corresponding IPLV and a RMD test to verify that simulation tests do this correctly and RDS will only verify IPLV.

Reply:
22-23 is updated to only check full load efficiency. Add new RDS Rule 22-37 for part load efficiency check.